bpf/process/types/basic.h

```diff
@@ -107,7 +107,7 @@ enum {
 	ACTION_SIGNAL = 9,
 	ACTION_TRACKSOCK = 10,
 	ACTION_UNTRACKSOCK = 11,
-	ACTION_NOTIFY_KILLER = 12,
+	ACTION_NOTIFY_ENFORCER = 12,
```

Comment:
is it okay @kkourt, I think we wanted to rename but this is technically a breaking change for API users now!

Reply:
It's  only internal, correct? So i think it should be fine.

Reply:
nevermind I confused myself I thought it was a protobuf enum, no worry then.